ASWF/meetings/tsc/2020-10-05.md

```diff
@@ -0,0 +1,128 @@
+      workspace is a good place to discuss, and any ideas an input are very 
+      appreciated.
+
+* Issue #1162 - PROCESS.md feedback:
+    - Thomas: In a good place. Would be great to have more people giving 
+      feedback. Explicit approval better than silent assent.
+    - Carol: Also think it looks good, but will have another look.
+    - Patrick: To highlight, there's a trend in OSS that saying nothing means 
+      its fine, but sometimes better to have explicit approval in a comment to 
+      an issue or PR. Some feedback to know where you stand.
+    - Michael: "LGTM" is a good response if you're in agreement.
+    - Patrick: If you take time to read it, take a time to comment.
+    - Thomas: On GH, approval is a click away also.
+    - Mark B: Only comment is with big changes throughout development. Since 
+      PRs were dependent on each other, decided to wait till work was more 
+      complete to review. Have some concerns that there's not enough time to 
+      vet work while pushing towards release date.
+    - Patrick: Good concern, and why we have 4 month stabilization period to 
+      test and dig into what we did. Up to you to tell us. We did our best to 
+      explain and add tests, which are also a way to explain the features. 
+      Please tell us if something is unclear or a concern.
+    - J: Understood. Sometimes huge wave of commits takes a lot of time to 
+      review. Good buffer to look and rethink it.
+    - Michael: Good to take time to review PRs if you can since it's easier to 
+      adjust work in progress than completed work. Probably too late to change
+      architectural decisions.
+    - Patrick: Best to start with unit tests and public API to get sense of 
+      features and changes. Can also start with Python. Start with something 
+      you are familiar with. 
+    - Thomas: Whenever not finding something in Python code, looking at C++ 
+      unit tests and that is very helpful.
+    - Mark B: Lots of stuff from previous TSC meetings and working group 
+      meetings where it's hard to track outcome of discussions. Not sure 
+      whether those changes were addressed or gone over. Need to check up on a 
+      lot of stuff and follow through. No issue with architectural stuff so 
+      much. Two or three things to follow up on though. Worried our concerns 
+      will be lost because of imposed deadline. Would be a shame if features 
+      go out if solutions doesn't match spirit of previous library version.
+    - Mark T: What things are you concerned about?
+    - Mark B: ICC display transforms. Live grading controls and how those 
+      interact. Environment resolution. Bit-depth - how it's expressed in graph 
+      and config.
+    - Patrick: Suggestion - build a list and next TSC meeting would like to 
+      discuss these points. We can address questions over several meetings. 
+      Could explain what we wanted to do and follow up on any concerns.
```

Comment:
Just to better explain my live comment: `Even if we did some WG sessions and created defects for all the features, we could still explain again what we all agreed to do and follow up on any concerns.`

Reply:
Updated.